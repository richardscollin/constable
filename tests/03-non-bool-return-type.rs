use constable::lookup;

#[lookup]
const fn foo(_x: u8) -> i8 {
    0
}

fn main() {}
