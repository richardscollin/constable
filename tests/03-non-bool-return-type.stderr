error: custom attribute panicked
 --> tests/03-non-bool-return-type.rs:3:1
  |
3 | #[lookup]
  | ^^^^^^^^^
  |
  = help: message: return type isn't one of bool, u8, u16, u32
